### PR TITLE
Remove response header from /authorize GET call

### PR DESCRIPTION
**Pull request by vahancouver**
Remove _accept: application/x-www-form-urlencoded_ header from /authorize endpoint for GET requests

I checked Android SDK. They don't send any response header.

### CODE REVIEW

**Review by george-bafaloukas-forgerock — APPROVED** [review]
Looks good mate. In case you have some time it might be worth adding a test on that

**Comment by vahancouver** [comment]
> Looks good mate. In case you have some time it might be worth adding a test on that

@george-bafaloukas-forgerock Added a unit test to check if the "Accept" header is nil

File: FRAuth/FRAuth/Config/OAuth2Client.swift [hunk 1]
```diff
@@ -378,7 +378,7 @@ public class OAuth2Client: NSObject, Codable {
         var header: [String: String] = [:]
         header[OpenAM.acceptAPIVersion] = OpenAM.apiResource21 + "," + OpenAM.apiProtocol10
         
-        return Request(url: self.serverConfig.authorizeURL, method: .GET, headers: header, urlParams:parameter, requestType: .urlEncoded, responseType: .urlEncoded, timeoutInterval: self.serverConfig.timeout)
+        return Request(url: self.serverConfig.authorizeURL, method: .GET, headers: header, urlParams:parameter, requestType: .urlEncoded, responseType: nil, timeoutInterval: self.serverConfig.timeout)
```
Comment (spetrov) [inline FRAuth/FRAuth/Config/OAuth2Client.swift:381]: Please update the copyright header of the file with current year

File: FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/FRAuth/FRAuthTests.swift [hunk 2]
```diff
@@ -530,4 +530,28 @@ class FRAuthTests: FRAuthBaseTest {
         // It should
         XCTAssertNotNil(FRAuth.shared)
     }
+    
+    func testOAuth2ClientAuthorizeCallDoesntContainAcceptHeader() {
```
Comment (spetrov) [inline FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/FRAuth/FRAuthTests.swift:534]: Update copyright header, please.

File: FRCore/FRCore/Network/Request.swift [hunk 3]
```diff
@@ -53,7 +53,7 @@ public struct Request {
     /// URL parameters as dictionary
     public let urlParams: [String: String]
     /// Response type as `ContentType`; "Accept" header will be automatically added
-    public let responseType: ContentType
+    public let responseType: ContentType?
```
Comment (spetrov) [inline FRCore/FRCore/Network/Request.swift:56]: Please, update the copyright header in the file

**Review by spetrov — CHANGES_REQUESTED** [review]
Changes look good to me! 
I have some minor comments to update the copyright headers of some files that have been updated.
Also, please, update the CHANGELOG.md with a note about this fix.

**Review by spetrov — APPROVED** [review]
👍🏻

**Pull request merged**